src/site/_includes/partials/post.njk

```diff
@@ -54,6 +54,32 @@
       </div>
     {% endif %}
 
+    {% if translation %}
+      {% if translation === true %}
```

Comment:
Instead of mixing types, what do you think about setting this to `translation: 'none'`?

Reply:
Yeah, I'm very confused about what `true` means here. I think "none" is better .

I also know we have test content for now but I presume we won't include English articles in other folders at launch.